app/code/Magento/Quote/Test/Unit/Model/QuoteManagementTest.php

```diff
@@ -122,6 +122,11 @@ class QuoteManagementTest extends \PHPUnit_Framework_TestCase
     protected $quoteMock;
 
     /**
+     * @var \Magento\Quote\Model\Quote\Validator\MinimumOrderAmount\ValidationMessage
+     */
+    protected $minimumAmountErrorMessage;
```

Comment:
We do not encourage inheritance based API's even in unit tests.
Please change this to `private`.